#!/bin/sh -e
## 40-path-correction.dpatch, by Steffen Joeris <steffen.joeris@skolelinux.de>
## and Luca Capello <luca@pca.it>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Correct the path for the usb_printerid program

if [ $# -lt 1 ]; then
echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
      -patch) patch $patch_opts -p1 < $0;;
      -unpatch) patch $patch_opts -p1 -R < $0;;
      *)
      echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
      exit 1;;
esac

exit 0

@DPATCH@
--- foo2zjs-20081007dfsg/hplj1000.ORG	2008-10-17 15:52:29.000000000 +0200
+++ foo2zjs-20081007dfsg/hplj1000	2008-10-17 15:52:47.000000000 +0200
@@ -47,7 +47,7 @@
 #    [snip]
 #    # Mount all other filesystems (except for NFS and /proc, which is already
 #
-PRINTERID=/bin/usb_printerid
+PRINTERID=`which usb_printerid`
 
 #
 #	Figure out how to log our messages
